warehouse/query-core/src/main/java/datawave/query/index/lookup/Union.java

```diff
@@ -85,7 +86,7 @@ public Union(Iterable<? extends IndexStream> streams) {
                 case UNKNOWN_FIELD:
                 case EXCEEDED_TERM_THRESHOLD:
                     // these nodes need to be persisted via the set of delayedNodes
-                    delayedNodes.add(stream.currentNode());
+                    delayedNodes.add(JexlNodes.wrap(stream.currentNode()));
```

Comment:
Same comment as above.  Also, keep in mind that the original code would have identified the reference node as the marked node, and the reference was the parent of the reference expression, so we didn't need to wrap anything with the existing code.  With the new code, since there are no reference nodes, and since AND nodes are now identified as the marked node, the AND node will need to be wrapped so that it doesn't mark the node it is added to if the tree is flattened.